Sources/SwiftKuery/Column.swift

```diff
@@ -84,8 +114,23 @@ public class Column: Field, IndexColumn {
         self.collate = collate
     }
     
-    /// Build the column using `QueryBuilder`.
-    ///
+    //MARK: Column Decription Functions
+    
+    /**
+     Function to build a String representation for referencing a `Column` instance.
```

Comment:
"Build a String representation to reference a `Column` instance." rather than "Function to build...etc"